CHANGELOG.md

```diff
@@ -88,6 +88,18 @@ Adding a new version? You'll need three changes:
   Note: this feature is experimental, and not available to translate other
   kubernetes objects that was originally supported.
   [#3935](https://github.com/Kong/kubernetes-ingress-controller/pull/3935)
+- Added `CombinedServices` feature gate that prevents the controller from
+  creating a separate Kong `Service` for each `netv1.Ingress` that uses
+  the same Kubernetes `Service` as its backend when `CombinedRoutes` feature
+  is turned on. Instead, the controller will create a single Kong `Service`
+  in such circumstances.
+  The feature is disabled by default to allow users to adapt to the changed
+  Kong Services naming scheme (`<namespace>.<service-name>.<port>` instead
+  of `<namespace>.<ingress-name>.<service-name>.<port>`) - it could break
+  existing monitoring rules that rely on the old naming scheme.
+  It will become the default behavior in the next minor release with the possibility
```

Comment:
```suggestion
  It will become the default behavior in the next major release with the possibility
```

?

Reply:
That's what we agreed upon when discussing the strategy for rolling such changes out at our team meeting: https://docs.google.com/document/d/1Ct-ZOceuXFUISPWrzQG8YMRCGPSVDe7RTdVQZ6uzq_U/edit#heading=h.1iupcjhqk605
```
Minor X -> off by default
Minor X+1 -> on by default, possibility to opt-out
Minor X+2 -> on by default, feature gate removal
```